fuel-tx/src/transaction/types/create.rs

```diff
@@ -172,6 +174,37 @@ impl Chargeable for Create {
         // is defined. Witness data should still be excluded.
         self.witnesses_offset()
     }
+
+    fn gas_used_by_metadata(&self, gas_costs: &GasCosts) -> Word {
+        let Create {
+            bytecode_witness_index,
+            witnesses,
+            storage_slots,
+            ..
+        } = self;
+
+        let contract: Option<Contract> = witnesses
+            .get(*bytecode_witness_index as usize)
+            .map(|c| c.as_ref().into());
```

Comment:
Should we just get the len here? Based on the type annotation above is seems like we're creating an owned copy of the contract which is going to add a lot more overhead to the gas calculation.

Reply:
Yep, I can update that in follow up PR.